### PR TITLE
Fix the Elixir sample code and update all dependencies

### PR DESCRIPTION
**Pull request by robsonpeixoto**

### CODE REVIEW

File: contrib/examples/elixir/lib/brod_sample/application.ex [hunk 1]
```diff
@@ -7,8 +7,11 @@ defmodule BrodSample.Application do
 
   def start(_type, _args) do
     children = [
-      BrodSample.GroupSubscriber
+      BrodSample.GroupSubscriberV2
+      # FIXME: If start both process only the "BrodSample.GroupSubscriberV2" consumes messages.
```
Comment (robsonpeixoto) [inline contrib/examples/elixir/lib/brod_sample/application.ex]: Any ideia of how to solve it?

Comment (robsonpeixoto) [reply contrib/examples/elixir/lib/brod_sample/application.ex]: It solved my problem: https://github.com/kafka4beam/brod/pull/522/commits/3f92dc69d7e0129fc2737cbfbdc02523bc5f50f6

File: contrib/examples/elixir/lib/brod_sample/group_subscriber.ex [hunk 2]
```diff
@@ -36,13 +37,14 @@ defmodule BrodSample.GroupSubscriber do
   end
 
   def handle_message(
```
Comment (GabriellaDiasA) [inline contrib/examples/elixir/lib/brod_sample/group_subscriber.ex:39]:
```suggestion

  @impl true
  def handle_message(
```

**Review by GabriellaDiasA — COMMENTED** [review]
I think `init/2` is also a callback, should probably add the `@impl true` line there as well

**Pull request closed without merging**